cclib/parser/orcaparser.py

```diff
@@ -1339,9 +1339,61 @@ def energy_intensity(line):
                     etoscs.append(float(intensity))
 
                     line = next(inputfile)
-
-                self.set_attribute('etenergies', etenergies)
-                self.set_attribute('etoscs', etoscs)
+                
+                # Some of these sections contain data that we probably do not want to be populating etenergies
+                # and/or etoscs with.  For example, the SOC corrected spectra are for mixed singlet/triplet states,
+                # so they do not correspond to the symmetries given in etsyms, and the energy values given are
+                # probably not what the user would expect to find in etenergies anyway?
+                # Also, there are twice as many SOC states as true spin states, so half of the etenergies wouldn't 
+                # have a symmetry in etsyms at all...
```

Comment:
Another technical limitation. If we end up wanting to support this, the question becomes one like [struct-of-arrays vs. array-of-structs](https://en.wikipedia.org/wiki/AoS_and_SoA). Do we have SOC as the outermost grouping, and energies/symmetries/multiplicities/etc. inside there, or those attributes as top-level groupings, and the SOC-corrected states inside each of those attributes? Or something in between?

We are leaning towards the latter. Both still require a unifying marker for "these are from the SOC-corrected states".

Reply:
Hmm yeah I think I see what you're getting at. In terms of style I think the second is more in-line with the current cclib style, but I might be wrong. In terms of whether it's useful I'm not sure, I've only ever come across SOC used to describe the degree of mixing between states, not the actual mixed states themselves, but my view is rather narrow. I suppose if it's there and easy to parse it would be a shame not to eventually support...